packages/instant/src/components/zero_ex_instant_container.tsx

```diff
@@ -28,6 +30,8 @@ export class ZeroExInstantContainer extends React.Component<{}, ZeroExInstantCon
         tokenSelectionPanelAnimationState: 'none' as SlideAnimationState,
     };
     public render(): React.ReactNode {
+        const PoweredByLogo = require('../assets/powered_by_0x.svg');
```

Comment:
No need to import this every render.

Reply:
I tried that at first, but I get the tslint complaint `require statement not part of an import statement [no-var-requires]`  if I put it outside of a function

Reply:
If I try to import, I get `TS2307: Cannot find module '../assets/powered_by_0x.svg'.`